plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/LibraryClasspathContainer.java

```diff
@@ -66,6 +67,6 @@ public String getDescription() {
 
   @Override
   public IClasspathEntry[] getClasspathEntries() {
-    return classpathEntries;
+    return classpathEntries.toArray(new IClasspathEntry[0]);
```

Comment:
I think this is why the arrays are used. A lot of the Eclipse API around this uses lists.